content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md

```diff
@@ -53,7 +53,10 @@ To push the Corda images:
     "corda-os-rest-worker" "corda-os-flow-worker"
     "corda-os-member-worker" "corda-os-p2p-gateway-worker"
     "corda-os-p2p-link-manager-worker" "corda-os-db-worker"
-    "corda-os-crypto-worker" "corda-os-plugins" )
+    "corda-os-flowMapper-worker" "corda-os-verification-worker"
```

Comment:
This should be `corda-os-flow-mapper-worker` and `corda-os-token-selection-worker`, likewise for `corda-ent-flow-mapper-worker` and `corda-end-token-selection-worker` below